Source/WebGPU/WGSL/AST/ASTFunctionDecl.h

```diff
@@ -41,7 +41,7 @@ class Parameter final : public Node {
 public:
     using List = UniqueRefVector<Parameter>;
 
-    Parameter(SourceSpan span, StringView name, UniqueRef<TypeDecl>&& type, Attribute::List&& attributes)
+    Parameter(SourceSpan span, const String& name, UniqueRef<TypeDecl>&& type, Attribute::List&& attributes)
         : Node(span)
         , m_name(WTFMove(name))
```

Comment:
Oops, forgot this `WTFMove` here and now `name` is a constant. So weird that local builds aren't failing though...